.pre-commit-config.yaml

```
@@ -21,5 +21,10 @@ repos:
 - repo: https://github.com/econchick/interrogate
   rev: 1.1.5
   hooks:
-   - id: interrogate
-     args: [-c, pyproject.toml]
+  - id: interrogate
+    args: [-c, pyproject.toml]
+- repo: https://github.com/terrencepreilly/darglint
+  rev: master
+  hooks:
+  - id: darglint
+    args: [-v 2]
```

``args: [-v 2]`` makes the error messages a bit less cryptic.

Without ``-v 2``:
```bash
functions.py:unionize_dataframe_categories:49: DAR401: -r TypeError
finance.py:convert_currency:409: DAR201: - return
```

With ``-v 2``:

```bash
functions.py:unionize_dataframe_categories:49: DAR401: Missing exception(s) in Raises section: -r TypeError
finance.py:convert_currency:409: DAR201: Missing "Returns" in Docstring: - return
```

---

Cool stuff! Could I suggest adding a comment here too?

```suggestion
    args: [-v 2]  # this config makes the error messages a bit less cryptic.
```